### PR TITLE
[mock_uss] migrate to uas_standards types

### PR DESCRIPTION
**Pull request by barroco**
This PR removes types from monitoringlib and uses the ones from uas_standards.

In addition, stack trace is now printed in case ValueError or unknown exceptions are returned.

### CODE REVIEW

File: monitoring/monitorlib/requirements.txt [hunk 1]
```diff
@@ -15,7 +15,8 @@ requests==2.25.1
 s2sphere==0.2.5
 shapely==1.7.1
 termcolor==1.1.0
-uas_standards==1.0.0
+uas_standards==1.1.0
+automated_testing
```
Comment (BenjaminPelletier) [inline monitoring/monitorlib/requirements.txt:19]: What is this?

**Review by BenjaminPelletier — COMMENTED** [review]
The changes look good, but why the deletion of ed269_test_sample_dataset.json and ed269_test.py?

File: monitoring/monitorlib/requirements.txt (hunk 1 above)
Comment (barroco) [reply monitoring/monitorlib/requirements.txt]: Inattention addition, my apologies.

**Comment by barroco** [comment]
> The changes look good, but why the deletion of ed269_test_sample_dataset.json and ed269_test.py?

They were moved to [uas_standards/tests](https://github.com/interuss/uas_standards/tree/main/tests)

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**